pkg/datasource/csvsource/tick_downloader_test.go

```diff
@@ -0,0 +1,47 @@
+package csvsource
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_Download_Binance_Default(t *testing.T) {
```

Comment:
I'd recommend disabling both tests to not strain those resources unnecessarily

Reply:
you can use an environment variable to enable it, like

if _, ok := os.LookupEnv("TEST_CSV_DOWNLOADER") ; !ok { t.Skip() }

Reply:
There doesn't seem to be an env file in the root of bbgo tho

Reply:
we're using dotenv, we run bbgo for development like this:

   dotenv .env.local -- go run ./cmd/bbgo ...

for testing, you can also do:

   TEST_CSV_DOWNLOADER=1 go test ./pkg/...

Reply:
thank you for the insight!